common/util.go

```diff
@@ -159,8 +163,33 @@ func GetDataLoc() (string, error) {
 	return GetLoc(data_dir)
 }
 
+func getAbsPath(subdir string) (dir string, err error) {
+	switch true {
```

Comment:
Seems like changing the previous semantic with contains.  "aetc" will now also return etc path, I don't think it is expected.